Packs/CiscoAMP/Integrations/CiscoAMP/CiscoAMP.py

```diff
@@ -0,0 +1,3535 @@
+        ))
+
+        if not raw_response_list[-1]['data']:
+            break
+
+    raw_response: Dict[str, Any] = combine_response_results(raw_response_list, pagination.is_automatic)
+
+    context_output = get_context_output(raw_response, ['links'])
+
+    readable_output = get_results_readable_output(raw_response)
+    readable_output += get_readable_output(
+        response=raw_response,
+        header_by_keys=ACTIVITY_HEADERS_BY_KEYS,
+        keys_to_items_option_1=['data'],
+        title=ACTIVITY_TITLE,
+    )
+
+    return CommandResults(
+        outputs_prefix='CiscoAMP.ComputerActivity',
+        outputs_key_field='connector_guid',
+        outputs=context_output,
+        readable_output=readable_output,
+        raw_response=raw_response,
+    )
+
+
+def computers_isolation_feature_availability_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Get information about available isolation options for a computer.
+
+    Args:
+        client (Client): Cisco AMP client to run desired requests
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about a computer's isolation options.
+    """
+    connector_guid = args['connector_guid']
+
+    try:
+        raw_response = client.computer_isolation_feature_availability_get_request(
+            connector_guid=connector_guid,
+        )
+        readable_output = get_isolation_options_readable_output(raw_response)
+
+    except DemistoException as exc:
+        # this is an expected behavior, when isolation is not allowed.
+        if exc.res and exc.res.status_code == HTTPStatus.METHOD_NOT_ALLOWED:
+            readable_output = 'Isolation is not allowed on policy.'
+
+        else:
+            raise  # if there's a different HTTP status code, it's not an expected behavior.
+
+    return CommandResults(
+        readable_output=readable_output
+    )
+
+
+def computer_isolation_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Get information about a computer's isolation.
+
+    Args:
+        client (Client): Cisco AMP client to run desired requests
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about a computer's isolation.
+    """
+    connector_guid = args['connector_guid']
+
+    raw_response = client.computer_isolation_get_request(
+        connector_guid=connector_guid,
+    )
+
+    readable_output = get_readable_output(
+        response=raw_response,
+        header_by_keys=ISOLATION_HEADERS_BY_KEYS,
+        keys_to_items_option_1=['data'],
+        title=ISOLATION_TITLE,
+    )
+    context_output = get_context_output(
+        response=raw_response,
+        contexts_to_delete=['links'],
+        item_to_add=('connector_guid', connector_guid)
+    )[0]
```

Comment:
is it never empty?

Reply:
Yes, the function will always return the current status of the isolation (isolated or not isolated).
The only scenario is that isolation isn't available, but in that case an error will pop-up 1st if the user hasn't followed the guide in the description (the error is: isolation not available on this policy).